build_tools/aws-sdk-code-generator/spec/interfaces/protocols_spec.rb

```diff
@@ -9,12 +9,13 @@
     }.to raise_error(/unsupported protocol/)
   end
 
+  # Prioritize JSON over CBOR due to better performance.
   context 'with smithy rpc protocol support' do
-    it 'selects smithy rpc v2 with priority' do
-      SpecHelper.generate_service(['ProtocolsRpcJson'], multiple_files: false)
-      client = ProtocolsRpcJson::Client.new(stub_responses: true)
-      expect(client.handlers).to include(Aws::RpcV2::Handler)
-      Object.send(:remove_const, :ProtocolsRpcJson)
+    it 'selects json with priority' do
+      SpecHelper.generate_service(['ProtocolsRpcJsonQuery'], multiple_files: false)
```

Comment:
The file name has the priority list. This implies rpc, json, query as the list. Please change to ProtocolsJsonRpcQuery.